group_vars/sovereign

```diff
@@ -0,0 +1,66 @@
+# required variables
+# ------------------
+# only two variables *must* be set:
```

Comment:
Isn't this a little misleading?  I think your intent is that a user reviews everything in this file and updates as needed.  Email passwords should be set, for example.